src/legacy/ui/public/chrome/api/injected_vars.ts

```diff
@@ -23,10 +23,6 @@ import { npSetup } from 'ui/new_platform';
 const newPlatformInjectedVars = npSetup.core.injectedMetadata;
 
 export function initChromeInjectedVarsApi(chrome: { [key: string]: any }) {
-  chrome.getInjected = (name?: string, defaultValue?: any) =>
-    cloneDeep(
-      name
-        ? newPlatformInjectedVars.getInjectedVar(name, defaultValue)
-        : newPlatformInjectedVars.getInjectedVars()
-    );
+  chrome.getInjected = (name: string, defaultValue?: any) =>
+    cloneDeep(newPlatformInjectedVars.getInjectedVar(name, defaultValue));
```

Comment:
Deleted the LegacyCore types, replaced with CoreStart/Setup were needed in the legacy code, so `getInjectedVars` is no longer present on the type. Had to adapt that and remove some legacy tests.

This is dead code anyway, and is going to be deleted in https://github.com/elastic/kibana/pull/76085